src/js/pyodide.js

```diff
@@ -162,12 +161,18 @@ function fixRecursionLimit() {
  * (This can be fixed once `Firefox adopts support for ES6 modules in webworkers
  * <https://bugzilla.mozilla.org/show_bug.cgi?id=1247687>`_.)
  *
- * @param {{ indexURL : string, fullStdLib? : boolean = true }} config
+ * @param {{ indexURL : string, fullStdLib? : boolean = true, input?: () => string, print?: (text: string) => void, printErr?: (text: string) => void }} config
  * @param {string} config.indexURL - The URL from which Pyodide will load
  * packages
  * @param {boolean} config.fullStdLib - Load the full Python standard library.
  * Setting this to false excludes following modules: distutils.
  * Default: true
+ * @param {undefined | (() => string)} config.input - Override the standard input callback. Should ask the user for one line of input.
+ * Default: undefined
+ * @param {undefined | ((text: string) => void)} config.print - Override the standard output callback.
+ * Default: undefined
+ * @param {undefined | ((text: string) => void)} config.printErr - Override the standard error output callback.
```

Comment:
How about calling these options `stdin`, `stdout`, and `stderr`?

Reply:
Sure, I can do that. I initially didn't do it, since what they're exposing is already a bit of a wrapper around Emscripten's `stdin`, `stdout`, `stderr`